Verification/FLiBe_Prop_Verification.py

```diff
@@ -0,0 +1,164 @@
+df12 = pd.DataFrame(data12, columns=['Temp','Vis'])
+data13 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Cohen_1956_69-31.csv'))
+df13 = pd.DataFrame(data13, columns=['Temp','Vis'])
+data14 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Gierszewski_1980_66-34.csv'))
+df14 = pd.DataFrame(data14, columns=['Temp','Vis'])
+data15 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Janz_1974_64-36.csv'))
+df15 = pd.DataFrame(data15, columns=['Temp','Vis'])
+data16 = pd.read_csv(Path(path + '/Flibe/Viscosity/Romatoski_Flibe_Viscosity_Cantor_1968_66-34.csv'))
+df16 = pd.DataFrame(data16, columns=['Temp','Vis'])
+
+
+fig2 = plt.figure(1, figsize=(10,8))
+ax = fig2.add_subplot()
+df9.plot(x='Temp', y='Vis', label='Abe 198 (67.2-32.8)', ax=ax, marker='^', linestyle='--', color='purple')
+df10.plot(x='Temp', y='Vis', label='Blanke 1956 (62.67-37.33)', ax=ax, marker='s', kind='scatter', color='dodgerblue')
+df11.plot(x='Temp', y='Vis', label='Blanke 1956 (69-31)', ax=ax, marker='x', kind='scatter', color='salmon')
+df12.plot(x='Temp', y='Vis', label='Cantor 1969 (64-36)', ax=ax, linestyle='-.', color='slategray')
+df13.plot(x='Temp', y='Vis', label='Cohen 1956 (69-31)', ax=ax, marker='o', linestyle='--', color='yellowgreen')
+df14.plot(x='Temp', y='Vis', label='Gierszewski 1980 (66-34)', ax=ax, linestyle=':', color='blue')
+df15.plot(x='Temp', y='Vis', label='Janz 1974 (64-36)', ax=ax, marker='+', kind='scatter', color='darkorange')
+df16.plot(x='Temp', y='Vis', label='Cantor 1968 (66-34)', ax=ax, linestyle='-', color='cyan')
+plt.plot(T2,Nu,'r--', label='Viscosity Used', linewidth=3, dashes=(10,20))
+plt.xlabel('Temperature - K')
+plt.ylabel('Viscosity - N/m^2/s')
+plt.title('Flibe Viscosity')
+plt.legend(loc='upper right')
+plt.grid()
+plt.show()
+
+Cp1 = np.zeros(steps)
+Cp2 = np.zeros(steps)
+Cp3 = np.zeros(steps)
+Cp4 = np.zeros(steps)
+Cp5 = np.zeros(steps)
+Cp6 = np.zeros(steps)
+Cp7 = np.zeros(steps)
+
+for i in range(0,steps):
+	Cp1[i] = Flibe_Cp_Verification_Data.Cp1(T2[i])
+	Cp2[i] = Flibe_Cp_Verification_Data.Cp2(T2[i])
+	Cp3[i] = Flibe_Cp_Verification_Data.Cp3(T2[i])
+	Cp4[i] = Flibe_Cp_Verification_Data.Cp4(T2[i])
+	Cp5[i] = Flibe_Cp_Verification_Data.Cp5(T2[i])
+	Cp6[i] = Flibe_Cp_Verification_Data.Cp6(T2[i])
+	Cp7[i] = Flibe_Cp_Verification_Data.Cp7(T2[i])
+
+fig3 = plt.figure(1, figsize=(10,8))
+plt.plot(T2,Cp,'r--', label='Specific Heat Capacity Used', linewidth=3, dashes=(10,20))
+plt.plot(T2,Cp1,'k--', label='2343')
+plt.plot(T2,Cp2,'c--', label='2347')
+plt.plot(T2,Cp3,'g--', label='2369')
+plt.plot(T2,Cp4,'m--', label='2380')
+plt.plot(T2,Cp5,'b--', label='2386')
+plt.plot(T2,Cp6,'y--', label='2390')
+plt.plot(T2,Cp7, linestyle='--', color='tab:brown',label='2414')
+plt.xlabel('Temperature - K')
+plt.ylabel('Specific Heat - J/kg-K')
+plt.ylim(2300, 2450)
+plt.title('Flibe Specific Heat Capacity')
+plt.legend(loc='upper right')
+plt.grid()
+plt.show()
```

Comment:
Remove this plt.show(), only put one at the end so all of the plots show up at once.